src/trusted/service_runtime/nacl_app_thread.c

```diff
@@ -117,7 +117,7 @@ struct NaClApp *NaClChildNapCtor(struct NaClApp *nap) {
     NaClXMutexUnlock(&nap_parent->children_mu);
   }
 
-  NaClAppInitialDescriptorHookup(nap_child);
+  //NaClAppInitialDescriptorHookup(nap_child);
```

Comment:
Is this intended?  If so, do you want to describe why you commented this but left the code?

Reply:
Very much appreciate the review/comments! This was just a mistake, I was testing moving the initial descriptor init from being a separate function to being part of the main fd transfer that's below. It worked, and then I forgot to delete.